### PR TITLE
PMM-4897: add instance branch clean

### PR DESCRIPTION
**Pull request by lunaticusgreen**

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-551-380897a
client docker - perconalab/pmm-client-fb:PR-551-380897a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-551-380897a.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-551-7b6e67b
client docker - perconalab/pmm-client-fb:PR-551-7b6e67b
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-551-7b6e67b.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-551-c7427ce
client docker - perconalab/pmm-client-fb:PR-551-c7427ce
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-551-c7427ce.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-551-bc749fb
client docker - perconalab/pmm-client-fb:PR-551-bc749fb
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-551-bc749fb.tar.gz

File: .gitmodules [hunk 1]
```diff
@@ -77,7 +77,7 @@
 [submodule "grafana-dashboards"]
 	path = sources/grafana-dashboards
 	url = https://github.com/percona/grafana-dashboards
-	branch = PMM-2.0
+	branch = add-instance
```
Comment (AlekSi) [inline .gitmodules]: Branch name should start with a ticket number.

Comment (lunaticusgreen) [reply .gitmodules]: Done

**Comment by AlekSi** [comment]
Closing in favour of https://github.com/Percona-Lab/pmm-submodules/pull/552.

**Pull request closed without merging**